### PR TITLE
Promo Site Projects Section

### LINKED ISSUE #149: [Promo V2] Projects Section
# Create the Past Projects section

## Desc
The past projects section is animated and has moving luggage showcasing the past projects

## Specs
* Last years winning projects are displayed on pieces of luggage a moving conveyor belt
* Contact us button on the billboard should trigger a mailto: hello@hackwestern.com
image

### PR DESCRIPTION
**Pull request by hunterchen7**
closes #149

# Quick Overview of Changes
- adds project section
image

# Checklist

- [x] If any frontend changes were made, include a screenshot/demo in the overview
- [x] Checked all uses of changed component(s)/function(s)
- [x] If any changes were made to our backend services, at least one (happy path) unit test was added OR a `TODO` comment was added to create one.
- [ ] Check that CI is passing.
- [ ] If no (re-)reviews after 1-2 days, ping the web leads on the discord to remind them to review the PR.

### CODE REVIEW

File: src/components/promo/projects.tsx [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            </Button>
+          </div>
+        </div>
+      </div>
+      <div className="w-full animate-small-bounce overflow-hidden">
+        <div className="-mb-1 flex animate-suitcase-slide gap-8 hover:[animation-play-state:paused] xl:gap-16 2xl:gap-20 3xl:gap-28 4xl:gap-36">
+          {
+            /* suitcases (projects) */
+            Array.from({ length: 3 }, () => listProjects)
+              .flat()
```
Comment (basokant) [inline src/components/promo/projects.tsx:86]: this should be outside the component/function body

File: src/components/promo/projects.tsx [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+      "https://d112y698adiu2z.cloudfront.net/photos/production/software_thumbnail_photos/002/682/847/datas/medium.png",
+    title: "SuperStage",
+    link: "https://devpost.com/software/superstage",
+  },
+  {
+    image:
+      "https://d112y698adiu2z.cloudfront.net/photos/production/software_photos/002/683/856/datas/medium.png",
+    title: "Infu",
+    link: "https://devpost.com/software/infu",
+  },
```
Comment (basokant) [inline src/components/promo/projects.tsx:240]: put these in a new file inside /constants/projects.ts

**Review by basokant — APPROVED** [review]

File: src/components/promo/projects.tsx (hunk 2 above)
Comment (hunterchen7) [reply src/components/promo/projects.tsx]: fixed

File: src/components/promo/projects.tsx (hunk 1 above)
Comment (hunterchen7) [reply src/components/promo/projects.tsx]: fixed

**Review by basokant — APPROVED** [review]

**Pull request merged**